internal/types/types_scan_result.go

```diff
@@ -59,10 +58,8 @@ func (r *ScanResult) SetOpenssl(info OpensslInfo) *ScanResult {
 }
 
 func (r *ScanResult) SetOS(info OSInfo) *ScanResult {
-	if errors.Is(info.Error, os.ErrNotExist) {
-		r.SetError(ErrDistributionFileMissing)
```

Comment:
These were the only two places using `ErrDistributionFileMissing`. We can probably consider removing the error from errors.go.

Reply:
I added this back in within a different spot. Thanks.